src/rhsm/connection.py

```diff
@@ -1866,26 +1866,24 @@ def bind(self, consumerId: str, entitle_date: datetime.datetime = None) -> List[
 
         return self.conn.request_post(method, description=_("Updating subscriptions"))
 
-    # FIXME: This method should return True, when it was successful, not None
-    def unbindBySerial(self, consumerId: str, serial: str) -> None:
+    def unbindBySerial(self, consumerId: str, serial: str) -> bool:
         """
         Try to remove consumed pool by serial number
         :param consumerId: consumer UUID
         :param serial: serial number of consumed pool
         """
         method = "/consumers/%s/certificates/%s" % (self.sanitize(consumerId), self.sanitize(str(serial)))
-        return self.conn.request_delete(method, description=_("Unsubscribing"))
+        return self.conn.request_delete(method, description=_("Unsubscribing")) is not None
```

Comment:
I need to check this again. I made this change before removing Restlib so this may no longer be the case where `None` is returned. This may also be a problem for this commit: 7bb6747ea since that was also changed before removing Restlib.